packaging/swift/update-ice-swift-nightly.sh

```diff
@@ -51,4 +54,4 @@ git config user.name "ZeroC"
 git config user.email "git@zeroc.com"
 git commit -m "ice: $ice_version Nightly build"
 git tag -a "$ice_version" -m "ice: $ice_version"
-git push origin main
+git push "https://x-access-token:${ICE_SWIFT_NIGHTLY_TOKEN}@github.com/zeroc-ice/ice-swift-nightly.git" main --tags
```

Comment:
Is this necessary? Does the remote not retain the token?

Reply:
I guess no.